info.plist

```diff
@@ -79,7 +79,13 @@
 				<key>runningsubtext</key>
 				<string>Searching docs for "{query}"...</string>
 				<key>script</key>
-				<string>/opt/homebrew/bin/php laravel.php "$1"</string>
+				<string>if [ -f "/opt/homebrew/bin/php" ]; then
+    /opt/homebrew/bin/php laravel.php "$1"
+elif [ -f "/usr/local/bin/php" ]; then
+    /usr/local/bin/php laravel.php "$1"
+elif [ -f "/usr/bin/php" ]; then
+    /usr/bin/php laravel.php "$1"
+fi</string>
```

Comment:
```suggestion
				<string>if [ -f "/opt/homebrew/bin/php" ]; then
    /opt/homebrew/bin/php laravel.php "{query}"
elif [ -f "/usr/local/bin/php" ]; then
    /usr/local/bin/php laravel.php "{query}"
elif [ -f "/usr/bin/php" ]; then
    /usr/bin/php laravel.php "{query}"
fi</string>
```

Reply:
@driftingly interesting…that change totally breaks it for me on the default setting “with input as argv”:

![Screen Shot 2021-11-08 at 13 39 40](https://user-images.githubusercontent.com/784333/140806979-9d4f7322-421f-4dbd-98d3-a911aa65c9bd.png)

If I change my workflow settings to “with input as {query}” then `{query}` works. However, changing [these lines](https://github.com/tillkruss/alfred-laravel-docs/blob/60d9f4d69442073f4298e39fdcd592f99162693c/info.plist#L83-L84):

```diff
				<key>scriptargtype</key>
-				<integer>1</integer>
+				<integer>0</integer>
```

Does that still work for you?

Reply:
Interesting, "with input as argv" works as expected with `"$1"`.  
Updating `scriptargtype` to 0 works with `"{query}"`.

Not sure why mine would have been different than yours, but I have been messing with the settings when it wasn't working.

Reply:
@tillkruss any preference on `$1` vs. `$query`?